metro.config.js

```diff
@@ -26,4 +27,4 @@ const config = {
     },
 };
 
-module.exports = mergeConfig(defaultConfig, expoConfig, config);
+module.exports = wrapWithReanimatedMetroConfig(mergeConfig(defaultConfig, expoConfig, config));
```

Comment:
It seems to me that it is also worth adding such a wrapper config metro, thanks to it errors reanimated on the console tell much more about what caused them.

In the case of this bug, this is what the bug looked like without the wrapper:

<img width="343" alt="image" src="https://github.com/user-attachments/assets/7eef0019-60eb-47cd-b484-6d1a205082b7" />

this is how it looked with the wrapper:

<img width="326" alt="image" src="https://github.com/user-attachments/assets/9223ca59-194e-46d7-a9ed-8e95270bd927" />